toolbox/certbot/renew_domain_with_certbot.sh

```diff
@@ -40,3 +40,5 @@ certbot/dns-route53 certonly -n --dns-route53 -d $DOMAIN
 cat /etc/letsencrypt/live/$DOMAIN/cert.pem > $ATD_AIRFLOW_HOMEDIR/haproxy/ssl/$DOMAIN.pem
 
 cat /etc/letsencrypt/live/$DOMAIN/privkey.pem >> $ATD_AIRFLOW_HOMEDIR/haproxy/ssl/$DOMAIN.pem
+
+chmod a+r $ATD_AIRFLOW_HOMEDIR/haproxy/ssl/$DOMAIN.pem
```

Comment:
cool, i had to look up `a+r`. so this makes the cert file readable by all users

Reply:
Yea - turns out we need this on the new systems -- my guess is that it's related to there not being overlap in the UIDs used on the RHEL vs the one's in the debian based docker images? This is a wild guess, but for whatever reason, it needs to be set.